packages/atomic/src/components/commerce/facets/atomic-commerce-facets/atomic-commerce-facets.tsx

```diff
@@ -0,0 +1,126 @@
+import {NumberValue, Schema} from '@coveo/bueno';
+import {
+  RegularFacet,
+  CategoryFacet,
+  DateFacet,
+  NumericFacet,
+  FacetGeneratorState,
+  buildProductListing,
+  buildSearch,
+  FacetGenerator,
+} from '@coveo/headless/commerce';
+import {Component, h, Element, Host, State, Prop} from '@stencil/core';
+import {
+  BindStateToController,
+  InitializableComponent,
+  InitializeBindings,
+} from '../../../../utils/initialization-utils';
+import {CommerceBindings as Bindings} from '../../atomic-commerce-interface/atomic-commerce-interface';
+
+/**
+ * The `atomic-commerce-facets` component automatically renders commerce facets based on the search query response.
+ * Unlike regular facets, which require explicit definition and request in the query, the `atomic-commerce-facets` component dynamically generates facets.
+ *
+ * @internal
+ */
+@Component({
+  tag: 'atomic-commerce-facets',
+  styleUrl: 'atomic-commerce-facets.pcss',
+  shadow: true,
+})
+export class AtomicCommerceFacets implements InitializableComponent<Bindings> {
+  @InitializeBindings() public bindings!: Bindings;
+  public facetGenerator!: FacetGenerator;
+  @Element() host!: HTMLElement;
+
+  /**
+   * The number of expanded facets inside the manager.
+   * Remaining facets are automatically collapsed.
+   *
+   * Using the value `0` collapses all facets.
+   * Using the value `-1` disables the feature and keeps all facets expanded. Useful when you want to set the collapse state for each facet individually.
```

Comment:
```suggestion
   * Using the value `-1` disables the feature and keeps all facets expanded.
```

I'm not sure how one would control each facet individually here (compared to classic atomic)